From a3092dd863e31df70d53514bb1765773b9f4540d Mon Sep 17 00:00:00 2001
From: Jiang Xin <zhiyou.jx@alibaba-inc.com>
Date: Tue, 30 Oct 2018 15:30:06 +0800
Subject: [PATCH] Agit release 6.6.0

Edit:

 * [Jiang Xin, 2022-09-02] Remove agit-release and agit-changelog, for
   we have patchwork and standalone CHANGELOG file in master branch.

Signed-off-by: Jiang Xin <zhiyou.jx@alibaba-inc.com>
Signed-off-by: Han Xin <hanxin.hx@alibaba-inc.com>
Signed-off-by: Xu Ziyang <yunhuai.xzy@alibaba-inc.com>
Signed-off-by: Jiuyang Xie <jiuyang.xjy@alibaba-inc.com>
---
 GIT-VERSION-GEN | 7 +++++++
 agit-version    | 1 +
 2 files changed, 8 insertions(+)
 create mode 100644 agit-version

diff --git a/GIT-VERSION-GEN b/GIT-VERSION-GEN
index 12f26c83d6..7b7a49598f 100755
--- a/GIT-VERSION-GEN
+++ b/GIT-VERSION-GEN
@@ -26,6 +26,13 @@ else
 	VN="$DEF_VER"
 fi
 
+if test -f agit-version
+then
+	AGITVN=$(cat agit-version | sed -e 's/-/./g')
+	AGITVN=$(expr "$AGITVN" : v*'\(.*\)')
+	VN="$VN.agit.$AGITVN"
+fi
+
 VN=$(expr "$VN" : v*'\(.*\)')
 
 if test -r $GVF
diff --git a/agit-version b/agit-version
new file mode 100644
index 0000000000..fc5c5419b3
--- /dev/null
+++ b/agit-version
@@ -0,0 +1 @@
+6.6.0
-- 
patchwork
